constants.py

```diff
@@ -28,4 +28,14 @@
 
 START_GAME = "Start Game"
 LEADER_BOARD = "Leader Board"
-EXIT_GAME = "Exit Game"
\ No newline at end of file
+EXIT_GAME = "Exit Game"
+MAIN_MENU = "Main Menu"
+
+def create_sprite_groups(number_of_groups: int = 1):
```

Comment:
getting these sprites into their own functions will help clean up our run file. Thanks for doing that